lxd/cgroup/init.go

```diff
@@ -64,6 +64,204 @@ func (info *Info) Mode() string {
 	return "unknown"
 }
 
+// Resources is a generic type used to abstract resource control features
+// support for the legacy and unified hierarchy.
+type Resources int
+
+const (
+	// Blkio resource control
+	Blkio Resources = iota
```

Comment:
Can you put in a line break after every const, will make parsing the associated comment a bit faster for my slow brain :)